src/content/app/tools/shared/components/tools-top-bar/ToolsTopBar.scss

```diff
@@ -3,7 +3,7 @@
 .topBar {
   min-height: 74px;
   background-color: $light-grey;
-  padding-left: 36px; // TODO: does this value repeat across Genome Browser, Entity Viewer, and Tools?
+  padding: 24px 10px 20px 30px; // TODO: does this value repeat across Genome Browser, Entity Viewer, and Tools?
```

Comment:
Is this still a TODO?

Reply:
I didn't add this comment so not sure but from what i can see this is now specific for blast and probably for tools. GB and entity viewer has different padding

Reply:
It is; we should figure out whether the distances in this component are the same across all apps. May be worth creating a ticket out of this.